doc/developer-guide/plugins/hooks-and-transactions/ssl-hooks.en.rst

```diff
@@ -159,6 +164,8 @@ TLS Inbound Hook State Diagram
 
      HANDSHAKE_HOOKS_PRE [shape=box];
      HANDSHAKE_HOOKS_PRE_INVOKE [shape=box];
+     HANDSHAKE_HOOKS_CLIENT_HELLO [shape=box];
+     HANDSHAKE_HOOKS_CLIENT_HELLO_INVOKE [shape=box];
```

Comment:
You'll want to add a section about the new hook in this file as well as it is set up for the other TS_SSL hooks.